apps/wallet-ui/src/lib/ecosystemConfig.ts

```diff
@@ -0,0 +1,34 @@
+import type { Chain } from "thirdweb";
+import { defineChain } from "thirdweb";
+
+type EcosystemIdentifier = `ecosystem.${string}` | string | undefined;
+
+const SPECIAL_ECOSYSTEM_CHAIN_IDS: Record<string, readonly number[]> = {
+  "mon-id": [1, 43114] as const,
```

Comment:
fine for now, but we could totally put this in the ecosystem config. right now we have only 'default chain', but we could let ppl add all the chains they care about